indigo_app/templates/indigo_api/_work_form_offcanvas_body.html

```diff
@@ -0,0 +1,19 @@
+{% load i18n %}
+
+<div class="offcanvas-header border-bottom" id="offCanvasWorkForm">
+  {% if form %}
+    <button
+      class="btn btn-success save"
+      type="submit"
+      form="edit-work-form"
+    >
+      {% if work.pk %}{% trans 'Save' %}{% else %}{% trans 'Create' %}{% endif %}
```

Comment:
I think for now there's no option of creating a new work using the offcanvas, but this'll be handy if and when that changes